undefined type error for composite literal expression when type is included via Used

```go
func TestIssue1151(t *testing.T) {
	type pkgType []int

	i := interp.New(interp.Options{})
	i.Use(interp.Exports{
		"pkg/pkg": map[string]reflect.Value{
			"Type": reflect.ValueOf((*pkgType)(nil)),
		},
	})
	i.ImportUsed()

	runTests(t, i, []testCase{
		{src: "pkg.Type{1}", res: "[1]"},
	})
}
```

Expected result: test succeeds.

Got:
```console
$ go test -run=TestIssue1151 ./interp/
--- FAIL: TestIssue1151 (0.00s)
    --- FAIL: TestIssue1151/#00 (0.00s)
        interp_eval_test.go:784: got an error: 1:28: undefined type
FAIL
FAIL    github.com/traefik/yaegi/interp 0.004s
FAIL
```

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -2186,7 +2186,10 @@ func (n *node) isType(sc *scope) bool {
 		suffixedPkg := filepath.Join(pkg, baseName)
 		sym, _, ok := sc.lookup(suffixedPkg)
 		if !ok {
-			return false
+			sym, _, ok = sc.lookup(pkg)
+			if !ok {
+				return false
+			}
 		}
 		if sym.kind != pkgSym {
 			return false

--- a/interp/interp_eval_test.go
+++ b/interp/interp_eval_test.go
@@ -1511,3 +1511,22 @@ func TestIssue1142(t *testing.T) {
 		{src: "a := 1; // foo bar", res: "1"},
 	})
 }
+
+func TestIssue1151(t *testing.T) {
+	type pkgStruct struct{ X int }
+	type pkgArray [1]int
+
+	i := interp.New(interp.Options{})
+	i.Use(interp.Exports{
+		"pkg/pkg": map[string]reflect.Value{
+			"Struct": reflect.ValueOf((*pkgStruct)(nil)),
+			"Array":  reflect.ValueOf((*pkgArray)(nil)),
+		},
+	})
+	i.ImportUsed()
+
+	runTests(t, i, []testCase{
+		{src: "x := pkg.Struct{1}", res: "{1}"},
+		{src: "x := pkg.Array{1}", res: "[1]"},
+	})
+}

